src/FlipMove.js

```diff
@@ -43,6 +44,12 @@ function getKey(childData: ChildData): Key {
   return childData.key || '';
 }
 
+function getElementChildren(children: ChildrenArray<Child>): Array<Element<*>> {
+  // Fix incomplete typing of Children.toArray
```

Comment:
Is this just a limitation of the Children type defined in React? Could you elaborate on the conditions required to remove the eslint-disable?

Reply:
`React.Children.toArray` removes `null`, `undefined`, and booleans from children set. The current [typing](https://github.com/facebook/flow/blob/master/lib/react.js#L231) for it is:
```
toArray<T>(children: ChildrenArray<T>): Array<$NonMaybeType<T>>;
```

This rules out `null` and undefined, but booleans are left